src/sentry/static/sentry/app/views/settings/projectAlerts/issueEditor/index.tsx

```diff
@@ -92,6 +93,7 @@ type State = AsyncView['state'] & {
   environments: Environment[];
   configs: {
     actions: IssueAlertRuleActionTemplate[];
+    filters: IssueAlertRuleActionTemplate[];
```

Comment:
I guess it doesn't really make any difference but `filters: IssueAlertConditionTemplate[]` might make more sense as filters more closely resemble conditions rather than actions (even though the types are identical)